tardis/transport/montecarlo/packet_trackers.py

```diff
@@ -99,14 +121,46 @@ def track(self, r_packet):
         ] = r_packet.last_interaction_type
         self.num_interactions += 1
 
+    def track_boundary_interaction(self, current_shell_id, next_shell_id):
```

Comment:
This

Reply:
This the only function defined. I have added that.